docs/vector.rst

```diff
@@ -199,7 +199,7 @@ Methods
         # c = [[1*4, 1*5, 1*6], [2*4, 2*5, 2*6]]
 
 .. note::
-    This has nothing in common with ``ti.cross``. ``a`` and ``b`` do not have to be 3 or 2 component vectors.
+    The outer product should not be confused with the cross product (``ti.cross``). For example, ``a`` and ``b`` do not have to be 2- or 3-component vectors for this function.
```

Comment:
Very nice organization of words!
**Only a bit curiousity**: Would it be a nice usage of hypens in `2- or 3-component`? :O

Reply:
I think the hyphen usage is correct. 

For example, you would say, "My backpack weighs nine pounds" or "I have a nine-pound backpack". The first sentence has no hyphen, because the adjective "nine" modifies the noun "pounds", but in the second sentence, "nine-pound" is an adjective modifying the noun "backpack." 

Similarly, "'a' and 'b' do not have to be vectors with 2 or 3 components." is written here as "'a' and 'b' do not have to be 2- or 3-component vectors."

Reply:
Thanks for you explanation!